tfjs-backend-webgpu/src/conv2d_mm_webgpu.ts

```diff
@@ -64,45 +66,29 @@ export class Conv2DMMProgram implements WebGPUProgram {
     if (hasPreluActivationWeights) {
       this.variableNames.push('preluActivationWeights');
     }
-    this.convInfo = convInfo;
     this.addBias = addBias;
     this.activation = activation;
     this.hasPreluActivationWeights = hasPreluActivationWeights;
 
-    [this.fitA, this.fitB] = this.getShapeFit();
-    this.shaderKey = `conv2DMM_${this.elementsPerThread}_${this.activation}_${
-        this.fitA}_${this.fitB}_${this.isChannelsLast}`;
-  }
-
-  getShapeFit(): boolean[] {
     const tileAOuter = this.workGroupSize[1] * this.elementsPerThread[1];
     const tileBOuter = this.workGroupSize[0] * this.elementsPerThread[0];
-    const tileInner = tileAOuter > tileBOuter ? tileAOuter : tileBOuter;
-    util.assert(
-        tileInner % this.workGroupSize[0] === 0 &&
-            tileInner % this.workGroupSize[1] === 0,
-        () =>
-            // tslint:disable-next-line: max-line-length
-        'tileInner must be multiple of workgroupsize.x and workgroupsize.y');
-    const tileSizeA = [tileAOuter, tileInner];
-    const tileSizeB = [tileInner, tileBOuter];
-    const dimAOuter = this.convInfo.outHeight * this.convInfo.outWidth;
-    const dimBOuter = this.convInfo.outChannels;
-    const dimInner = this.convInfo.filterHeight * this.convInfo.filterWidth *
-        this.convInfo.inChannels;
-
-    return [
-      tilesFitEvenlyIntoShape(tileSizeA, [dimAOuter, dimInner]),
-      tilesFitEvenlyIntoShape(tileSizeB, [dimInner, dimBOuter])
-    ];
+    this.tileInner = 32;
```

Comment:
In fact, `tileInner` has no relationship with `tileAOuter` and `tileBOuter`. It can be any value as long as it satisfies these conditions https://github.com/tensorflow/tfjs/pull/6411/files#diff-1b2285212e86bc2038bd788d3ab61aaa6675de1baa1ee87316ea1b69bfaca359R55. To keep the original default value, I set 32 here. I can add a TODO here so that we can adjust this value based on input size(dimInner) in future.